fertility/orderly-web.yml

```diff
@@ -90,7 +90,7 @@ web:
   image:
     repo: vimc
     name: orderly-web
-    tag: master
+    tag: mrc-2951
```

Comment:
I deployed the ncov one with `ab37e7a` which is the same thing as you have here.